bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooworkspace.py

```diff
@@ -33,10 +33,26 @@
 */
 """
 
+from ._utils import _kwargs_to_roocmdargs
+
 
 class RooWorkspace(object):
+    def __init__(self, *args, **kwargs):
+        # Redefinition of `RooWorkspace` constructor for keyword arguments.
+        # The keywords must correspond to the CmdArg of the constructor function.
+        args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
+        self._init(*args, **kwargs)
+
+    def __call__(self, obj_data):
+        # To enable `RooWorkspace` instance to behave as a function.
+        # Object is passed to the instance to use the general `RooWorkspace.obj()` function.
+        return getattr(self, "obj")(obj_data)
+
     def Import(self, *args, **kwargs):
         """
         Support the C++ `import()` as `Import()` in python
         """
+        # Redefinition of `RooWorkspace.Import()` for keyword arguments.
+        # The keywords must correspond to the CmdArg of the `Import()` function.
+        args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
         return getattr(self, "import")(*args, **kwargs)
```

Comment:
```suggestion
def RooWorkspace_import(self, *args, **kwargs):
    # Redefinition of `RooWorkspace.import()` for keyword arguments.
    # The keywords must correspond to the CmdArg of the `import()` function.
    args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
    return self._import(*args, **kwargs)


setattr(RooWorkspace, "import", RooWorkspace_import)
```
The `RooWorkspace::Import` keyword argument pythonization is not so straight-forward, otherwise you would have done it before, right? :) What you suggest here doesn't work, have you tried the `rf511_wsfactory_basic.py` tutorial?

To make it work, you actually have the define a Pythonization for `RooWorkspace.import`, because `RooWorkspace.Import` is already a wrapper function generated by cppyy to avoid having a function with the keyword name `import`.

Still, you can't directly define a function called `import` in the RooWorkspace mirror class because `import` is a Python keyword, However, you can define the function with a different name and then use `setattr` to monkey-patch it to RooWorkspace, as suggested above.